db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+
+for user_n in 0..10
+  user = User.new(name: 'user_'+user_n.to_s, email: 'user_'+user_n.to_s+'@example.com')
```

Comment:
We must use either double quotes or single quotes everywhere. What do you think if we used double quotes everywhere?
```suggestion
  user = User.new(name: "user_" + user_n.to_s, email: "user_" + user_n.to_s + "@example.com")
```